kymatio/scattering2d/frontend/torch_f.py

```diff
@@ -0,0 +1,178 @@
+__all__ = ['Scattering2D']
+
+import torch
+from ..backend import cdgmm, Modulus, SubsampleFourier, fft, Pad, unpad, NAME
+from ..filter_bank import filter_bank
+from ..utils import compute_padding
+from ..scattering2d import scattering2d
+from ...frontend.torch_f import Scattering
+
+if NAME != 'torch' and NAME != 'skcuda':
+    raise (RuntimeError('The only supported backend by the torch frontend are torch and skcuda.'))
+
+class Scattering2D(Scattering):
+    """Main module implementing the scattering transform in 2D.
```

Comment:
Fix justification of the body text here. Probably best to fit it to 80 characters wide. (Does PEP8 say anything about this?)

Reply:
Do you know an automatic way to switch to 79 characters wide? (like, I don't see myself doing it manually for the whole pkg)

Reply:
In vim it's `set tw=79`, then it wraps as you type. You can type `gq` to reformat old lines. Most other editors have something similar.

Reply:
Well, for the future!